tests/trading/test_cancelOrder.py

```diff
@@ -105,8 +108,14 @@ def test_exceptions(contractsFixture):
     assert(cash.approve(makeOrder.address, fix('10000'), sender = tester.k1) == 1), "Approve makeOrder contract to spend cash"
     makerInitialCash = cash.balanceOf(tester.a1)
     marketInitialCash = cash.balanceOf(market.address)
-    orderID = makeOrder.publicMakeOrder(orderType, fxpAmount, fxpPrice, market.address, outcomeID, tradeGroupID, sender=tester.k1)
-    assert(orderID != 0), "Order ID should be non-zero"
+
+    # yesShareToken = contractsFixture.applySignature('shareToken', market.getShareToken(YES))
+    # print yesShareToken.balanceOf(tester.a1)
+    # print "Making order"
+    orderID = makeOrder.publicMakeOrder(orderType, fxpAmount, fxpPrice, market.address, outcomeID, ZEROED_ORDER_ID, ZEROED_ORDER_ID, tradeGroupID, sender=tester.k1)
+    # print yesShareToken.balanceOf(tester.a1)
```

Comment:
Delete dead code rather than leaving it commented out.

Reply:
Sorry, I had meant to remove this, but I missed it before committing.